runtime/doc/eval.txt

```diff
@@ -1813,7 +1813,7 @@ getcmdpos()			Number	return cursor position in command-line
 getcmdtype()			String	return current command-line type
 getcmdwintype()			String	return current command-line window type
 getcurpos()			List	position of the cursor
-getcwd()			String	the current working directory
+getcwd( [{scope}] )			String	the current working directory
```

Comment:
@HiPhish I meant that you should remove space here as well, because this is how other lines look like. Exactly because beautification of documentation is not the aim of the PR; should it be needed it is easier to replace all such cases at once.